src/content/docs/data-apis/understand-data/new-relic-data-types.mdx

```diff
@@ -59,6 +53,8 @@ Metrics are a strong solution for storing data long-term, and understanding tren
 
 Conceptually, "metrics" is a broad, general category. There are various ways New Relic measures and reports metrics but, in practice, when using the New Relic UI, you usually won't have to understand how exactly this happens. In our documentation, we typically will just refer to "metrics," regardless of how that data is reported, unless there's a reason you need to know more (like understanding [how to query your data](/docs/using-new-relic/data/understand-data/query-new-relic-data)).
 
+For a list of the metrics we collect, see [our documentation on metrics](/docs/telemetry-data-platform/ingest-manage-data/understand-data/metric-data-type).
```

Comment:
I don't think this is technically correct. that doc contains the types of metrics within the `Metric` data type. It's not including the other types of metrics we collect (e.g., metric timeslice data, which is still our main metric data source for APM, Browser, and Mobile, and not the metrics reported via events, like for infrastructure; and I'm not sure the types of metric data in this doc (gauge, count, and such) count as a 'list of metrics', as I'd maybe expect a 'list of metrics' to be more about the specific types of data). If  we're going to point to that doc, we might want to point to it from the 'dimensional metrics' entry. I'm not sure of the goal we're trying to accomplish with this addition.

Reply:
This was an existing sentence from the doc that I re-arranged; happy to just delete it.